README.md

```diff
@@ -183,9 +184,64 @@ When this plugin is installed, custom metrics will be created automatically for
 * SNS
 * Scheduled Events
 
-### Profiler Plugin
+### Logging Plugin
+
+The IOpipe agent comes bundled with a logging plugin that allows you to attach IOpipe to the `logging` module.
+
+Here's an example of how to use the logging plugin:
+
+```python
+from iopipe import IOpipe
+from iopipe.contrib.logging import LoggingPlugin
+
+iopipe = IOpipe(plugins=[LoggingPlugin()])
 
-**Note:** This feature is still in beta. Want to try it out? Find us on [Slack](https://iopipe.now.sh).
+@iopipe
+def handler(event, context):
+    context.iopipe.log.info('Handler has started execution')
```

Comment:
what if a user just does print() to stdout?

Reply:
@ewindisch I added a `redirect_stdout` option.